src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -138,27 +138,30 @@ def on_time_step(self, event: Event):
         move those households to a new address
         """
         households = self.population_view.get(event.index)
-        household_heads = households.loc[
-            households["relation_to_household_head"] == "Reference person"]['household_id']
-
-        households_that_move_idx = self.randomness.filter_for_rate(
-            household_heads.index, self.household_move_rate(household_heads.index)
+        household_ids = households.loc[
+            households["relation_to_household_head"] == "Reference person"
+        ]["household_id"]
```

Comment:
You may want to consider shortening the string constants you're passing around. Doing string comparisons on long strings is needless expense, never mind the additional memory it'll take in the population table.

Even better, you may want to put strings like "Reference person" into a constant in data_values.py, so you redefine to a different string without fuss.